lib/composer.json

```diff
@@ -0,0 +1,33 @@
+{
+		"name": "wcpay/wcpay-lib",
+		"description": "A package for hiding re-nameed dependencies.",
```

Comment:
> Typo: re-nameed

Thanks, that's fixed now.

> Also, it might be good to fix the tabs vs spaces and some other such inconsistency in this and the root composer.json file.

I was actually battling with this in earlier commits, but it got out of focus. Now I switched to 4 spaces for indentation in both files (https://github.com/Automattic/woocommerce-payments/pull/6862/commits/5bf37999d183523864fb9878350b6e4002c5ecde), as Composer uses spaces. A bit more changes in the PR, but at least we'll finally have those files well indented.